wpilibj/src/test/java/edu/wpi/first/wpilibj/geometry/GeometryTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.wpi.first.wpilibj.geometry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GeometryTest {
```

Comment:
In general, we have one test class per real class.  Can we split up this file to keep with this standard?

Also, test classes can be package private.

Reply:
Should I do the same for the C++ tests? i.e. splitting up the tests for each geometry class and then splitting each test into a method?

Reply:
Split up the tests and made classes package-private in 7fc4c4f. Let me know if I should break up the tests similarly for C++.

Reply:
Yes please for C++.

Reply:
Split up test classes for C++ in a3c6fed